### DISCUSSION TITLE
allow `IBM I: Go to file...` to open member when not found

### DISCUSSION THREAD

**Discussion by m-tyler** [codefori/vscode-ibmi#769] [IBM i] [answered]
I genuinely like the command `IBM I: Go to file...` 

I USE IT ALL THE TIME.

It does have one lacking feature, which I only bring up because of a little bit of ignorance on my part.  I was searching for a member, it was not in the list, so in the empty input box (also no results appear), I just started typing the member to open as I do with the command `IBM I: Open File`.  When I press Enter nothing happens.  It might seem obvious to you those who wrote the command `IBM I: Go to file...` but if I was not closely paying attention I am wondering why the open command is no longer working. 

So if I type a member path in the `IBM I: Go to file...` input can it open a member like `IBM I: Open File` command?

Thanks

**Comment by worksofliam** [comment]
Good idea. I'd need to look into if the API can support this functionality. Then perhaps we can merge this functionality together.

**Reply by m-tyler** [reply]
Thanks. No rush from my perspective as I have learned my lesson about the two commands and it's a simple ESC key to be able to trigger off the other.

**Comment by m-tyler** [comment, marked as answer]
this feature request has been implemented and it works wonderfully.
